### PR TITLE
Group aup and add/remove aup for admin configuration

### PR DESCRIPTION
**Pull request by cgeorgilakis**
https://jira.argo.grnet.gr/browse/RCIAM-1099
Need email addition.

### CODE REVIEW

File: src/main/resources/META-INF/group-management-changelog.xml [hunk 1]
```diff
@@ -221,5 +221,26 @@
...
+
+        <addColumn tableName="USER_VO_GROUP_MEMBERSHIP">
+            <column name="IS_ADMIN" type="boolean"  defaultValueBoolean="false">
+                <constraints nullable="false"/>
+            </column>
+        </addColumn>
+        <addColumn tableName="GROUP_AUP">
+            <column name="URL" type="VARCHAR(255)"/>
+        </addColumn>
+        <dropNotNullConstraint tableName="GROUP_AUP" columnName="CONTENT" columnDataType="BLOB"/>
```
Comment (laskasn) [inline src/main/resources/META-INF/group-management-changelog.xml:234]: perfect!

File: src/main/java/org/keycloak/plugins/groups/jpa/entities/UserVoGroupMembershipEntity.java [hunk 2]
```diff
@@ -45,6 +50,8 @@ public class UserVoGroupMembershipEntity {
     @Column(name="JUSTIFICATION")
     protected String justification;
 
+    @Column(name="IS_ADMIN")
```
Comment (laskasn) [inline src/main/java/org/keycloak/plugins/groups/jpa/entities/UserVoGroupMembershipEntity.java:53]: Nice addition!

**Review by laskasn — APPROVED** [review]
Looks good! Let's merge

**Comment by cgeorgilakis** [comment]
Email examples:

- You have added as VO admin to the group test
- You have removed as VO admin from the group test

**Pull request merged**